tutorials/nbconverted/general_usage.py

```diff
@@ -0,0 +1,227 @@
+
+# In[19]:
+
+
+# update compartment names and strata
+strata = ["Image_Metadata_Well", "Image_Metadata_Plate"]
+compartments = ["Per_Cells", "Per_Cytoplasm", "Per_Nuclei"]
+
+# Updating linking columns for merging all compartments
+linking_cols = {
+    "Per_Cytoplasm": {
+        "Per_Cells": "Cytoplasm_Parent_Cells",
+        "Per_Nuclei": "Cytoplasm_Parent_Nuclei",
+    },
+    "Per_Cells": {"Per_Cytoplasm": "Cells_Number_Object_Number"},
+    "Per_Nuclei": {"Per_Cytoplasm": "Nuclei_Number_Object_Number"},
+}
+
+
+# Now that we have stored the updated the parameters, we can use them as inputs for SingleCells class to proceed with the merging of all the NF1 sqlite tables into a single consolidated dataset.
+
+# In[20]:
+
+
+# setting up sqlite address
+sqlite_address = f"sqlite:///{str(plate_data)}"
+
+# loading single cell morphology data into pycyotminer's SingleCells Object
+single_cell_profile = SingleCells(
+    sql_file=sqlite_address,
+    compartments=compartments,
+    compartment_linking_cols=linking_cols,
+    image_table_name="Per_Image",
+    strata=strata,
+    merge_cols=["ImageNumber"],
+    image_cols="ImageNumber",
+    load_image_data=True,
+)
+
+# mering all sqlite table into a single tabular dataset (csv)
+sc_profile = single_cell_profile.merge_single_cells(
+    sc_output_files=sc_profile_path, compression_options="gzip"
+)
+
+# saving single-cell morphology dataset
+sc_profile.to_csv(sc_profile_path, compression="gzip")
+
+# displaying dataset
+sc_profile.head()
+
+
+# Now that we have created our merged single-cell profile, let's move on to the next step: loading our `platemaps`. 
+# 
+# Platemaps provide us with additional information that is crucial for our analysis. They contain details such as well positions, genotypes, gene names, perturbation types, and more. In other words, platemaps serve as a valuable source of supplementary information for our single-cell morphology profile.
+
+# In[21]:
+
+
+# loading plate map and display it
+platemap_df = pd.read_csv(plate_map)
+platemap_df.head(8)
+
+
+# ## Annotation
+# 
+# In this step of the walkthrough, we will combine the metadata with the merged single-cell morphology dataset. To accomplish this, we will utilize the `annotation` function provided by `pycytominer`.
+# 
+# The `annotation` function takes two inputs: the merged single-cell morphology dataset and its associated plate map. By combining these two datasets, we will generate an annotated_profile that contains enriched information.
+# 
+# More information about the `annotation` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.annotate)
+# 
+
+# In[22]:
+
+
+# annotating merged single-cell profile with metadata
+annotated_df = annotate(
+    profiles=sc_profile,
+    platemap=platemap_df,
+    join_on=["Metadata_well_position", "Image_Metadata_Well"],
+)
+
+# save annotated profile
+annotated_df.to_csv(anno_profile_path, compression="gzip")
```

Comment:
Like above, should you use the pycytominer `output` function instead that is embedded in the annotate, normalize, and feature select functions to output the files in one line instead of using pandas `to_csv`?